app/models/season/phase_switcher.rb

```diff
@@ -1,9 +1,20 @@
+# frozen_string_literal: true
+
 class Season::PhaseSwitcher
 
   def self.season
     Season.current
   end
 
+  def self.destined(phase)
+    if phase.in?(whitelisted_phases)
+      self.public_send(phase)
+    end
+  end
+
```

Comment:
Again personal preference, but these double blank lines before `private` are kinda oldskool and hurt my eye 😉  - why not just a single blank line before and after?